pkg/cmd/pulumi/destroy.go

```diff
@@ -238,6 +250,9 @@ func newDestroyCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVarP(
 		&debug, "debug", "d", false,
 		"Print detailed debugging output during resource operations")
+	cmd.PersistentFlags().BoolVar(
+		&remove, "remove", false,
```

Comment:
The requested flag `--rm` is more than one ASCII character and so won't work. `r` is currently mapped to `--refresh`. Therefore opt to use no shorthand and use `--remove`.